packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -203,6 +204,16 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
     });
   };
 
+  // Get rid of click/drag/pointer/mousedown handlers if selection isn't allowed so that text selection can happen
+  if (!allowsSelection) {
+    pressProps = {
```

Comment:
Think you need to update this one as well?

Reply:
ah derp, thanks for catching